exchange/bitswap/workers.go

```diff
@@ -9,16 +11,31 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 16
+
+func init() {
+	twc := os.Getenv("IPFS_TASK_WORKERS")
+	if twc != "" {
+		n, err := strconv.Atoi(twc)
+		if err != nil {
+			return
```

Comment:
okay error + default value SGTM.

maybe we should endeavor to not do anything bad on `init()` calls? only validation of program, etc?

could also move this to main once main is back to reasonable